tests/test_gcloud.py

```diff
@@ -91,7 +92,7 @@ def test_open_write(self, MockBlob):
         # File data is not actually written until close(), so do that.
         f.close()
 
-        MockBlob().upload_from_file.assert_called_with(tmpfile)
+        MockBlob().upload_from_file.assert_called_with(tmpfile, content_type=mimetypes.guess_type(self.filename)[0])
```

Comment:
Here it would be preferable to put `content_type` on the next line. This will still make the line longer than 79 characters but that's OK.